src/ui/EditTask.svelte

```diff
@@ -147,7 +170,21 @@
                 : '',
             dueDate: task.dueDate ? task.dueDate.format('YYYY-MM-DD') : '',
             doneDate: task.doneDate ? task.doneDate.format('YYYY-MM-DD') : '',
+            reminderTime: task.reminderTime
+                ? task.reminderTime.format('YYYY-MM-DD HH:mm')
+                : '',
         };
+        //Only load reminder panel if activated in settings
+        const { reminder } = getSettings();
+        if (!reminder) {
```

Comment:
Why `!reminder` - why the negation?

Oh, the comment says 'only load' but I think that what the code is doing suggests that the comment should say something like `Remove the reminder panel if not activated in settings`